omnigan/transforms.py

```diff
@@ -117,12 +117,12 @@ def __call__(self, data):
 
 class Normalize:
     def __init__(self, opts):
-        if opts.data.normalization == "default":
```

Comment:
@tianyu-z this formulation introduces a loophole if opts don't have `opts.data.normalization`.

In general be careful with not doing `if/elif` without `else` or you might expose the code to leaky logic with unhandled cases.

Here if the default behaviour you want is `((0.485, 0.456, 0.406), (0.229, 0.224, 0.225))` then it should be in the else statement as I have done here